### ISSUE TITLE
Excuse me, have you trained mars dataset by yourself？

### ISSUE THREAD

**Issue by chrenlily** [xiaoxiong74/Object-Detection-and-Tracking#2]
How do you train other types of Mars dataset？

**Comment by xiaoxiong74** [comment]
> How do you train other types of Mars dataset？

I didn't train other types of Mars dataset, I just reused the original deep_sort model(just trained for person)

**Comment by chrenlily** [comment]
So you don't have the deep appearance descriptor of  bicycle?

**Comment by xiaoxiong74** [comment]
yes,  when tracking other objects, kalman filtering is considered more in deep_sort model.

**Comment by xiaoxiong74** [comment]
> So you don't have the deep appearance descriptor of bicycle?

@chrenlily  yes, when tracking other objects, kalman filtering is considered more in deep_sort model.

**Comment by chrenlily** [comment]
ok! When you track a bicycle, just think about detection and kalman filtering.

**Comment by chrenlily** [comment]
can I ask your another question? Why is the total number of bicycles and pedestrians less than the number on the box?

**Comment by zzc-ai** [comment]
> > 您如何训练其他类型的火星数据集？
> 
> 我没有训练其他类型的火星数据集，我只是重用了原始的deep_sort模型（仅针对人员进行训练）

您的代码是什么环境运行的呢，我是win一直报错，如果看到的话希望得到您的指导
